Gemfile.lock

```diff
@@ -67,8 +68,8 @@ GEM
     faraday (0.13.1)
       multipart-post (>= 1.2, < 3)
     ffi (1.9.18)
-    globalid (0.3.7)
-      activesupport (>= 4.1.0)
+    globalid (0.4.2)
+      activesupport (>= 4.2.0)
     govuk_elements_rails (3.0.2)
```

Comment:
could we update this as well, while we're at it? there are some bug fixes (especially for Firefox)

Reply:
I updated to latest (3.1.3). It depends on `govuk_frontend_toolkit (>= 6.0.2)` which I also updated to. That broke the build, so I rolled back to investigate at another time.

Reply:
ok, no worries. let's do it in another PR ;)